puppet-chrome/index.ts

```diff
@@ -82,8 +90,6 @@ const defaultArgs = [
   '--allow-running-insecure-content',
 
   '--lang=en-US,en;q=0.9',
-  '--window-size=1920,1080',
-  '--window-position=0,0',
```

Comment:
Why did you get rid of these?

Reply:
I figured it was kind of weird to set those there since we're setting screensize in code now for every request.

Reply:
Agreed